pages/pricing/index.tsx

```diff
@@ -8,8 +8,13 @@ import { ArrowLeftCircleIcon, ArrowRightCircleIcon } from "@heroicons/react/24/s
 import "slick-carousel/slick/slick.css";
 import "slick-carousel/slick/slick-theme.css";
 import { pricingData } from "@/utils";
+import { useRouter } from "next/router";
 
 const Pricing: NextPage = () => {
+  const router = useRouter();
+  useEffect(() => {
```

Comment:
why route this page back home??

Reply:
yes they said we should do that because of our soft launch since the page isnt ready 
directives is from geo and the project manager  
@sammy-code98